Expensify/App
# Update desktop icon

cc @shawnborton 

### Details
Updates the icon used for the desktop app

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/153043

### Tests
1. Comment out [publish and notarize steps in `electon.config.js`](https://github.com/Expensify/Expensify.cash/pull/681#issue-507153860) 
2. Run `npm run desktop-build`
3. Verify desktop app compiled in `dist/` folder has new icon

### Tested On


### Screenshots
#### Desktop
<img width="1164" alt="Screen Shot 2021-02-08 at 9 49 46 AM" src="https://user-images.githubusercontent.com/2838819/107253541-a750ec80-69f3-11eb-96f9-95164080019e.png">
<img width="396" alt="Screen Shot 2021-02-08 at 9 50 13 AM" src="https://user-images.githubusercontent.com/2838819/107253546-a7e98300-69f3-11eb-8b2e-53a7085a9b3d.png">
